taichi/transforms/cfg_optimization.cpp

```diff
@@ -6,14 +6,16 @@
 TLANG_NAMESPACE_BEGIN
 
 namespace irpass {
-void cfg_optimization(IRNode *root) {
+void cfg_optimization(IRNode *root, bool after_lower_access) {
   TI_AUTO_PROF;
   auto cfg = analysis::build_cfg(root);
   while (true) {
     bool modified = false;
     cfg->simplify_graph();
     if (cfg->unreachable_code_elimination())
       modified = true;
+    if (!after_lower_access && cfg->store_to_load_forwarding())
```

Comment:
Maybe we can still do local store forwarding after lower_access?

Reply:
Yes, then I think we'll have to pass `bool after_lower_access` to `store_to_load_forwarding()` and even `reaching_definition_analysis()`...

Reply:
Oh I've just done it in this PR.

Reply:
OK sounds good. No problem. Thanks!